src/Makefile

```diff
@@ -39,8 +37,8 @@ $(PROJECT_NAME).bin : $(PROJECT_NAME).elf
 		@ 	$(OBJCOPY) -O ihex $< $(PROJECT_NAME).hex
 		@ @
 
-$(PROJECT_NAME).elf : $(OBJ)
-	$(CC) $(INCLUDES_DIRECTORIES)   -std=gnu99 -mabi=$(ABI)  $(OPT) -march=$(ARCH) -nostartfiles -msmall-data-limit=8 -msave-restore -Os -fmessage-length=0 -fsigned-char -ffunction-sections -fdata-sections -fno-common -w -Xlinker --gc-sections  -Wl,-Map,$(PROJECT_NAME).map,$(LDFLAGS) --specs=nano.specs --specs=nosys.specs  -T  Link.ld  -o $@  $^
+$(PROJECT_NAME).elf : $(OBJ) Link.ld
+	$(CC) -std=gnu99 -mabi=$(ABI)  $(OPT) -march=$(ARCH) -nostartfiles -msmall-data-limit=8 -msave-restore -Os -fmessage-length=0 -fsigned-char -ffunction-sections -fdata-sections -fno-common -w -Xlinker --gc-sections  -Wl,-Map,$(PROJECT_NAME).map,$(LDFLAGS) --specs=nano.specs --specs=nosys.specs  -T  Link.ld  -o $@  $^
```

Comment:
Hi, and thanks.
That actually doesn't repro for me, but this should fix it. That'll pass the list of .o objects to the linkers instead of the things that caused the project to be rebuilt. Changing Link.ld SHOULD cause a relink, but shouldn't be passed in the list of objects.
Oh, and I added a .gitignore at no extra charge. :-)